compute/admin_guide/configure/agentless-permission-templates.adoc

```diff
@@ -0,0 +1,68 @@
+== Agentless Permission Templates
+
+Agentless scanning requires specific permissions to work on your self-hosted deployments.
+Different platforms have different permission requirements. 
+To make applying the right permissions easier, you can download permission templates for your specific platforms.
+The following sections explain the different permission templates and what they entail.
+
+=== AWS
+
+You need to download three permission templates as JSON files for AWS cloud account scanning. The templates are needed to enable the different ways accounts are scanned: independent scanning, scan hub account, and target account.
+
+==== Independent Scanning
```

Comment:
@rcaballeromx @Aqsaxvv092 I understand that "Independent" means that we include the permissions for hub and target together and it is for scanning within each cloud account.

Is this related to the "individual account" and "multiple account" phrasing in the configure agentless scanning topic? If not, how does it map for our users?